### DISCUSSION TITLE
Histogram Bins

### DISCUSSION THREAD

**Discussion by morancoder** [Data-Centric-AI-Community/fg-data-profiling#964] [Q&A]
Hey everyone,

I am using pandas profiling and have an issue with the way histogram presented. I can't figure out how to exclude outliers from my plot. Right now because of the outliers, most of the observations fall in one or two bins. Is there any way to exclude extreme values when plotting the histogram in html? Can I get finer bins in the middle of the plot rather than tails?
Thank you for your help.

Moran

**Comment by sbrugman** [comment]
Hi @morancoder, good question. The histogram binning is currently uniform, for performance reasons. Dynamic binning (using bayesian blocks) has been implemented in the past, but does not scale to most datasets and became a bottleneck in the profiling.

Feature suggestions for alternatives are welcomed, provided that the solution is performant (e.g. threshold-based or alternative plotting).
